raw-svo.setup.lua

```diff
@@ -309,6 +309,38 @@ signals.gmcpcharvitals:connect(function()
   end
 end)
 #end
+
+#if class == "monk" then
+signals.gmcpcharvitals:connect(function()
+  if gmcp.Char.Vitals.charstats then
+    for index, val in ipairs(gmcp.Char.Vitals.charstats) do
+      local stance = val:match("^Stance: (%w+)$")
+      local form = val:match("^Form: (%w+)$")
+
+      if stance then
+        me.path = "tekura"
+        me.form = nil
+        me.stance = stance:lower()
+        sk.ignored_defences['shikudo'].status = true
```

Comment:
Yeah, that's probably a good idea. I've updated both sections to swap appropriately.